proxmox/config_qemu.go

```diff
@@ -113,6 +114,10 @@ func (config ConfigQemu) CreateVm(vmr *VmRef, client *Client) (err error) {
 		params["bios"] = config.Bios
 	}
 
+	if config.EFIDisk != "" {
+		params["efidisk"] = config.EFIDisk
```

Comment:
Thats should be efidisk0, did you check the api?

https://pve.proxmox.com/pve-docs/api-viewer/index.html#/nodes/{node}/qemu

Reply:
Guess you are right.
I will double this.